man/org.freedesktop.home1.xml

```diff
@@ -101,6 +101,8 @@ node /org/freedesktop/home1 {
               out h send_fd);
       @org.freedesktop.systemd1.Privileged("true")
       ReleaseHome(in  s user_name);
+      InhibitSuspendHome(in  s user_name,
```

Comment:
"Suspend" seems the wrong word. the operation you block here we otherwise call "locking".

maybe call this InhibitLockOnSleepHome() or so? because that's what this really does, inhibit the automatic locking at sleep time.

Reply:
or hmm, the please-suspend/dont-suspend also uses the word "suspend", so maybe ignore me? dunno. naming is hard...

Reply:
ah that was me tweaking the documentation a bit. The docs called it suspending fairly consistently when I first wrote the code (though the function was always called Lock). I've since made the docs call it locking throughout

The pam module still calls it suspending, as does the code in various places. Do you want me to rename all those places too?